### PR TITLE
NA-27/Add-add-note-functionality

### PR DESCRIPTION
**Pull request by yevgeny-semak**

### CODE REVIEW

File: src/pages/notes.page.jsx [hunk 1]
```diff
@@ -9,17 +10,44 @@ import apiInstance from "../api.instance";
...
+            .then((response) => {
+                setNotes((prevValue) => [response.data.note, ...prevValue]);
+            })
+            .catch((error) => {
+                console.log('Error occured: ' + error);
+            });
+    };
 
+    const deleteNote = (id) => {
+        apiInstance.delete('http://localhost:8000/api/notes/' + id)
```
Comment (veranika-zakharchenia) [inline src/pages/notes.page.jsx:42]: could you create file _config.json_ where you can store all constants?
f.e.
_config.json_

> {
>     "apiHost": "http://localhost:8000",
> }

in other files

> const { apiHost } = require('../config.json');

> apiInstance.post(`${apiHost}/api/notes/`, {

File: src/pages/notes.page.jsx [hunk 2]
```diff
@@ -9,17 +10,44 @@ import apiInstance from "../api.instance";
 const Notes = () => {
     const [notes, setNotes] = useState(null);
     const [filteredNotes, setFilteredNotes] = useState(notes);
+    const history = useHistory();
 
     useEffect(()=>{
-        apiInstance.get('http://localhost:8000/api/notes/')
+        apiInstance.get('http://localhost:8000/api/notes/', { validateStatus: status => { return status < 500 }})
```
Comment (veranika-zakharchenia) [inline src/pages/notes.page.jsx:16]: interesting approach :)
does this catch positive and negative scenarios? 500 is also negative as 401

File: src/pages/notes.page.jsx (hunk 1 above)
Comment (yevgeny-semak) [reply src/pages/notes.page.jsx]: done

File: src/pages/notes.page.jsx (hunk 2 above)
Comment (yevgeny-semak) [reply src/pages/notes.page.jsx]: refactored to be using exact codes

**Review by veranika-zakharchenia — APPROVED** [review]

**Pull request merged**